app/javascript/packages/webauthn/is-webauthn-passkey-supported.ts

```diff
@@ -0,0 +1,30 @@
+type IsWebauthnPasskeySupported = () => boolean;
+
+const MINIMUM_IOS_VERSION = 16;
+
+const MINIMUM_ANDROID_VERSION = 9;
+
+function isQualifyingIOSDevice(): boolean {
+  const match = navigator.userAgent.match(/iPhone; CPU iPhone OS (\d+)_|iPad; CPU OS (\d+)_/);
```

Comment:
The `|iPad` is confusing to me, none of the test user agents have `iPad` in there, should we add one to check?

Reply:
> The `|iPad` is confusing to me, none of the test user agents have `iPad` in there, should we add one to check?

On closer inspection, apparently this is based on outdated assumptions about the iPad user agent, and newer versions of iPad use a user agent more similar to macOS. I see some leads which suggest using [`maxTouchPoints`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints) to differentiate, but maybe it's worth just removing from the initial launch. I'll remove for now but raise it with the team.